### ISSUE TITLE
[A11Y][List item] List item with external indicator should have a role link

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#918]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### OS Version

26

### App version

3.0.0 (723)

### Describe the issue

Clickable list item has a button role by default. But when the indicator corresponds to an external link or action, then it should have a link role

### Expected Behavior

Link role only when external indicator is activated

**Comment by pya35** [comment]
fixed on build 729
